src/main/java/reposense/model/FileTypeManager.java

```diff
@@ -3,13 +3,17 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.logging.Logger;
+
+import reposense.system.LogsManager;
 
 /**
  * {@code FileTypeManager} is responsible for holding a list of whitelisted formats and user-specified custom
  * groupings.
  */
 public class FileTypeManager {
     private static final String DEFAULT_GROUP = "other";
+    private static final Logger logger = LogsManager.getLogger(FileTypeManager.class);
```

Comment:
Do you plan to use this logger somewhere?

Reply:
I forgot to remove the logger. I added coz I wanted to give a warning message if the group already exists in the list, sth like `Skipping group as %s already in repository %s %s`. But then I notice fileTypeManager does not store the repo location and branch name. So if this warning message is not that important I feel I should not let fileTypeManager access additional information?